stl/inc/type_traits

```diff
@@ -615,11 +615,22 @@ struct has_unique_object_representations : bool_constant<__has_unique_object_rep
 _EXPORT_STD template <class _Ty>
 _INLINE_VAR constexpr bool has_unique_object_representations_v = __has_unique_object_representations(_Ty);
 
+#if 1 // TRANSITION, DevCom-10201896, LLVM-59002, EDG's __is_aggregate isn't suitable for LWG-3823

```

Comment:
For now, let's treat DevCom-10201896 as applying to both MSVC and EDG:
```suggestion
#if 1 // TRANSITION, DevCom-10201896, LLVM-59002
```
(If EDG remains unfixed when MSVC is, we can replace with an EDG-specific VSO number.)